pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -127,6 +134,18 @@ func (n *NodeLabeller) Execute() bool {
 	return true
 }
 
+// GetSupportedCPUmodels returns slice of supported cpus on the node. Slice doesn't contain
+// cpus which are in the obsoleteCPUs list
+func (n *NodeLabeller) GetSupportedCPUmodels() []string {
```

Comment:
Instead of copying the whole list possibly dozens of time per second, how about adding a `IsCPUModelSupported` method, which looks up the model from a map?